src/mawpy/constants.py

```diff
@@ -0,0 +1,14 @@
+USER_ID = 'user_ID'
+STAY = 'stay'
+STAY_DUR = 'stay_dur'
+STAY_LAT = '             stay_lat'
```

Comment:
Are the spaces intentional? If so, can you add a comment explaining why?

Reply:
also for `orig_lat` below

Reply:
The spaces are intentional, as the input CSV files had these spaces for these two columns. 
I will clean up the constants for column names once the ThinkLab team confirms #24

Reply:
I meant adding a comment in the code :), otherwise people without the context will find it vert strange.